confgenerator/config.go

```diff
@@ -442,6 +442,16 @@ type LoggingNetworkReceiver interface {
 	GetListenPort() uint16
 }
 
+func (m *loggingReceiverMap) GetListenPorts() map[string]uint16 {
```

Comment:
Optional: Since this is now an exported function intended to be used out of this confgenerator, we might want to add a short comment. 
```suggestion
// GetListenPorts returns a map of receiver IDs to ports for all LoggingNetworkReceivers
func (m *loggingReceiverMap) GetListenPorts() map[string]uint16 {
```

Reply:
This is great! Committed your suggestion.